kubectl-rook-ceph.sh

```diff
@@ -245,6 +249,28 @@ function run_rook_cr_status() {
   fi
 }
 
+function run_purge_osd() {
+  if [ "$#" -eq 2 ] && [ "$2" = "--force" ]; then
```

Comment:
To avoid the command duplication, how about setting a var for the force flag? For example:
```suggestion
  if [ "$#" -eq 2 ] && [ "$2" = "--force" ]; then
    force_removal=true
  else
    force_removal=false
  fi
```
Then there is a single command that uses the var

Reply:
updated